### PR TITLE
Show recording and snapshot storage separately if different sizes

### LINKED ISSUE #5062: Storage Overview -> Data - Split Snapshots and Recordings usage
**Describe what you are trying to accomplish and why in non technical terms**
I want to be able to see how much space my snapshots take and how much is available for them and I want to be able to see how much my recordings take and how much space is available for them since I keep the snapshots locally and recordings on a NAS.

**Describe the solution you'd like**
I'd like two rows, one for snapshots and the other for recordings if the locations are different

**Additional context**
Currently this information is not correct since recordings are saved to a NAS
image

```
volumes:
      - /home/user/frigate/config/config.yml:/config/config.yml:ro
      - /home/user/frigate/media/clips:/media/frigate/clips
      - /home/user/frigate_nas_share:/media/frigate/recordings
```

### PR DESCRIPTION
**Pull request by NickM-27**
Closes #5062

### CODE REVIEW

File: web/src/routes/Storage.jsx [hunk 1]
```diff
@@ -18,95 +18,118 @@ export default function Storage() {
...
   const { service } = stats || initialStats || emptyObject;
 
+  if (!service || !storage) {
+    return <ActivityIndicator />;
+  }
+
+  let storage_usage;
+  if (
+    service &&
+    service['storage']['/media/frigate/recordings']['total'] != service['storage']['/media/frigate/recordings']['total']
```
Comment (T3chArmy) [inline web/src/routes/Storage.jsx:28]: This if statement appears to be checking if it is not equal to itself. What's the advantage of not displaying both all of the time?

Comment (T3chArmy) [reply web/src/routes/Storage.jsx]: so does this only give the value of the used/total for the drive itself, not the individual folders?

Comment (NickM-27) [reply web/src/routes/Storage.jsx]: Yes, otherwise it would require spinning up the drive every time stats were checked which is not okay.

Comment (T3chArmy) [reply web/src/routes/Storage.jsx]: That makes sense. Thanks for the clarification!

**Review by blakeblackshear — APPROVED** [review]

**Pull request merged**